cli/contract_validation.js

```diff
@@ -0,0 +1,57 @@
+import { Project } from "ts-morph";
+import path from "path";
+import chalk from "chalk";
+
+const validateContract = async (contractPath) => {
+  const project = new Project();
+  project.addSourceFilesAtPaths(contractPath);
+  const baseFileName = path.parse(contractPath).base;
+  const contractClassFile = project.getSourceFile(contractPath);
+  const contractClasses = contractClassFile.getClasses(baseFileName);
+  for (const contractClass of contractClasses) {
+    const classStructure = contractClass.getStructure();
+    const { decorators, properties } = classStructure;
+    const hasBindgen = decorators.find(
+      (decorator) => decorator.name === "NearBindgen"
+    );
+    if (hasBindgen) {
+      const constructors = contractClass.getConstructors();
+      const hasConstructor = constructors.length > 0;
+      const propertiesToBeInited = properties.filter((p) => !p.initializer);
+      // reson: examples/clean-state.js
+      if (!hasConstructor && propertiesToBeInited.length === 0) {
+        return true;
+      }
+      if (!hasConstructor && propertiesToBeInited.length > 0) {
+        console.log(
+          chalk.redBright(
+            `Ops, constructor isnt initialized, after initialization include ${propertiesToBeInited
```

Comment:
We are not initializing the constructor, we are initializing parameters inside the constructor.
I think something like "All parameters must be initialized in the constructor" is closer to what we want to say here.